### DISCUSSION TITLE
Any way to crop videos?

### DISCUSSION THREAD

**Discussion by OldGeezer916** [smplayer-dev/smplayer#171] [Q&A]
I just switched to Smplayer a few months ago from VLC. In every way but one Smplayer is far superior. 

I like to watch in full screen16:9. It's easy in VLC to crop in to eliminate the black bars, whether it's an old series in 5:4 or a movie in Panavision. I have experimented in Smplayer & found on some videos you can set the aspect & uncheck the black borders & fill the screen. Problem is it stretches the video way out of aspect. VLC just completely chops the top & bottom or sides.

I am attaching captures so you can see the difference I am talking about.

If there isn't a way in Smplayer to crop, could you consider adding that in a future update?

![Panavision screenshot](https://user-images.githubusercontent.com/74526703/128644542-41b032c8-2f24-4eba-a942-b55f09c898ce.jpg)
![Smplayer 16 9](https://user-images.githubusercontent.com/74526703/128644551-d29aed2f-aec9-4104-97ad-65fb65508e35.png)
![VLC fullscreen crop](https://user-images.githubusercontent.com/74526703/128644557-aa6cdebc-9170-4c9f-9b49-adc5fbc64168.png)

**Comment by OldGeezer916** [comment]
Problem mostly resolved under issues: https://github.com/smplayer-dev/smplayer/issues/172

**Comment by jimmy-1000** [comment]
> I just switched to Smplayer a few months ago from VLC. In every way but one Smplayer is far superior.
> 
> I like to watch in full screen16:9. It's easy in VLC to crop in to eliminate the black bars, whether it's an old series in 5:4 or a movie in Panavision. I have experimented in Smplayer & found on some videos you can set the aspect & uncheck the black borders & fill the screen. Problem is it stretches the video way out of aspect. VLC just completely chops the top & bottom or sides.
> 
> I am attaching captures so you can see the difference I am talking about.
> 
> If there isn't a way in Smplayer to crop, could you consider adding that in a future update?
> 

But if your crop the video you may lost something! Many people complain of Netflix for crop to 16:9 all episodes of Seinfeld (original 4:3)

**Reply by OldGeezer916** [reply]
> But if your crop the video you may lost something! Many people complain of Netflix for crop to 16:9 all episodes of Seinfeld (original 4:3)

**Comment by OldGeezer916** [comment]
@jimmy-1000:
For some things that is true. Sometimes with older TV series I do watch in default 5:4 aspect. It's easy to just hot key & turn off cropping. 

I watched the cropped Seinfield episodes. When Netflix does this they don't just do a straight centered crop. They produce it & adjust to the scene & move up or down when needed. I didn't see any time watching the whole series where heads were being cut off or anything else noticeable. 

In movies & most TV series that are in wide Panavision the extra space on the sides is usually entirely unnecessary. Maybe it's great sitting in a theater watching that extra wide screen with the sides showing what is all the way in the 2 adjacent counties, but on a 16:9 TV you would rarely miss a thing cropping off the sides. It's not like taking off the top & bottom of an old TV series.
